### ISSUE TITLE
Collect and plot NTP information

### ISSUE THREAD

**Issue by igarny** [perfsonar/graphs#36]
Hi guys,

It would be very useful to be able to correlate the information / results from NTP daemon for the stability of the clock to the latency graphs.

Latency measurements are very much reliant on the stable clocks and looking at latency graphs the user has to assume things, which might not be correct.

We really need to see between the precision or stability of the clock and I suggest we get an expert opinion about it. But first lets agree we need to have that monitored.

Best regards,
Ivan

**Comment by apertome** [comment]
Hi Ivan,

We already have plans to do just that, though I don't know if they're tracked in a ticket, so we'll keep this one open. I'm setting it to 3.6 for now, although I don't know if it will happen on that timeframe or not.

**Comment by igarny** [comment]
Hi Michael,

Thanks!
There are nevertheless some tough points as well. The stability is a function of the 2 servers participating in the test. We need to find a way to collect and expose that information. 
It becomes more like a research project!

Best regards,
Ivan

**Comment by apertome** [comment]
Indeed, the plan is to stick the NTP data in Esmond and it can then be plotted like any other test result.

**Comment by igarny** [comment]
How did you decide to expose the info below?
"delay":"0.03421","stratum":"2","address":"62.40.123.115","dispersion":"0.13933","offset":"-0.000881"

I mean, I am missing sys_jitter (the overall assessed jitter) here.

**Comment by igarny** [comment]
Hi Michael, 

This is just a suggestion:
Checkout this page: https://www.eecis.udel.edu/~mills/ntp/html/monopt.html.
I believe the interesting part is with **peerstats**

**Comment by mfeit-internet2** [comment]
The other thing we can do is develop a test/tool pair to report this stuff, have pScheduler run it at whatever interval we like and archive the results to esmond.
